cloud/blockstore/libs/storage/volume/volume_actor.cpp

```diff
@@ -670,6 +670,30 @@ void TVolumeActor::HandleServerDestroyed(
     OnServicePipeDisconnect(ctx, msg->ServerId, now);
 }
 
+void TVolumeActor::HandleGetDrTabletInfoResponse(
+    const TEvDiskRegistryProxy::TEvGetDrTabletInfoResponse::TPtr& ev,
+    const NActors::TActorContext& ctx)
+{
+    Y_UNUSED(ctx);
+    const auto* msg = ev->Get();
+
+    if (HasError(msg->GetError())) {
+        constexpr TInstant DRTabletIdRequestRetryInterval =
```

Comment:
please move it outside of the function and class
similar to how it's done here https://github.com/ydb-platform/nbs/blob/8d8cdc34ca22b227aef86313f87dbb075723bfb4/cloud/blockstore/libs/storage/ss_proxy/path_description_backup.cpp#L23